RadixWallet/Clients/TransactionClient/Models/TransactionModels.swift

```diff
@@ -190,7 +190,7 @@ extension TransactionClient {
 
 // MARK: - ManifestReviewRequest
 public struct ManifestReviewRequest: Sendable {
-	public let manifestToSign: TransactionManifest
+	public let manifestToSign: RawTransactionManifest
```

Comment:
I think `manifestToSign` ought to be change to `uncheckedManifest: UncheckedManifest` or `unvalidatedManifest: UnvalidatedManifest` and possibly we can rename `ManifestReviewRequest` to `ManifestToReviewRequest` ?